stacks/stacks-v2.yaml

```diff
@@ -495,6 +495,79 @@ stacks:
       - name: supersetSecretKey
         description: Superset's secret key used to generate e.g. user session tokens
         default: supersetSecretKey
+  end-to-end-security:
+    description: >-
+      A stack used to demonstrate an end-to-end security concept.
+
+      Please note that this stack is tightly coupled with the demo.
+      So if you install the stack you will get demo-specific parts (such as Keycloak users or regorules).
```

Comment:
Should the demo specific stuff move into the demo? Or is that difficult?

Reply:
The thing is that is is very hard or even impossible. The hdfs and trino rego rules are targeted to the demo (e.g. table names and columns), but already needed in the stack etc.

Reply:
I assumed the OPA rules, and table generation could be done in the demo (rather than the stack). Maybe I'm missing something.

Reply:
Exactly, the tables are created in the demo. The thing is that Trino (and HDFS) are not working without rego rules, so the stack needs to contain *some* rego rules. We could deploy a allow-all rego rule in the stack and override it in the demo with a specific one.
But all of this introduces complexity, and I'm not sure if it is wort it, as 95% will probably install the demo and not only the stack. But this is just an estimation, I don't have actual numbers.
Another example is that the stack needs already needs to know where Impersonation is used and where not (e.g. it's a hdfs config if hive is allowed to impersonate others). And the `krb5` principals already have `KNAB.COM` in their name, I guess there is no way around this.

As an alternative we could have an empty stack and install everything in the demo, WDYT of that?

Reply:
Ah ok, I understand.
Yeah, could move it all to demo, but that could lead to further confusion.

I would say just leave it as is, so I'll mark this as resolved.